youtube_dl/extractor/gaskrank.py

```diff
@@ -0,0 +1,50 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from re import sub
```

Comment:
In youtube-dl usually standard modules are imported with simply the syntax 'import xxx'